auth/src/main/java/com/firebase/ui/auth/util/CredentialsApiHelper.java

```diff
@@ -31,7 +31,8 @@
  * A {@link com.google.android.gms.tasks.Task Task} based wrapper for the Smart Lock for Passwords
  * API.
  */
-public class CredentialsApiHelper {
+@Deprecated
+public class CredentialsApiHelper implements CredentialApiHelper {
```

Comment:
One last (annoying) comment: it's pretty confusing to have these two names only differ by an 's' in the middle.  Let's change the name of the interface?  Maybe `CredentialTaskApi` or something like that.